live-examples/css-examples/opacity.html

```diff
@@ -0,0 +1,32 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">opacity: 0;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_one">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice" initial-choice="true">
```

Comment:
`initial-choice` should only be set for one of the choices. Or you can omit it entirely, and it will select the first one - but in this case you probably don't want to do that :).

Reply:
oh I didn't checked this one when duplicating the "unit" snippet of HTML though it's pretty obvious when you see it ^- ^ I'll amend the contribution guidelines to add this :)